### ISSUE TITLE
Some ETA for RGP 1.5 announced at GDC?

### ISSUE THREAD

**Issue by oscarbg** [GPUOpen-Tools/Radeon-GPUProfiler#24]
at GDC was said to be coming in April..

**Comment by ahosier** [comment]
Hi oscarbg,

RGP 1.5 is all ready to go! We're just waiting on the release of a compatible driver which should happen within a week or two. Thanks for getting in touch.

Tony.

**Comment by oscarbg** [comment]
this is good news!
thanks Tony.

**Comment by ahosier** [comment]
Hi oscarbg,

We just released RGP V1.5.1, which has all of the GDC features in.

Thanks,
Tony.

**Comment by oscarbg** [comment]
thanks @ahosier will take a look at it soon!
also just seeing on twitter today (https://twitter.com/SebAaltonen/status/1128538223557398528)
"Radeon GPU Analyzer doesn't have SM 6.2 support"
is that true? any RGP docs pointing up to what shader model 6.x is supported? and can share future plans on closing gap of supported shader model versions (currently up to SM6.4)..
really no real or urgent need right now, just curiosity :-)
can open a separate issue if prefered..

**Comment by oscarbg** [comment]
maybe RGP supports SM6.2 after all.. as I read too fast and it's "Radeon GPU Analyzer" which may have some limitations..
